metanorma/Dockerfile.in

```diff
@@ -2,76 +2,21 @@ FROM ${ROOT_IMAGE}
 
 MAINTAINER Open Source at Ribose <open.source@ribose.com>
 
+# install dependencies
 RUN apt-get update && apt-get install -y curl gnupg2
-
 RUN curl -sSL https://raw.githubusercontent.com/metanorma/metanorma-linux-setup/master/ubuntu.sh | bash -s
 
-# RVM version to install
-ARG RVM_VERSION=stable
-ENV RVM_VERSION=${RVM_VERSION}
-
-# RMV user to create
-ARG RVM_USER=metanorma
-ENV RVM_USER=${RVM_USER}
-
-# Install + verify RVM with gpg (https://rvm.io/rvm/security)
-RUN curl -sSL https://get.rvm.io | bash -s
-RUN cat /usr/local/rvm/gemsets/global.gems
-RUN echo "bundler" >> /usr/local/rvm/gemsets/global.gems \
-    && echo "rvm_silence_path_mismatch_check_flag=1" >> /etc/rvmrc \
-    && echo "install: --no-document" > /etc/gemrc
-
-# Workaround tty check, see https://github.com/hashicorp/vagrant/issues/1673#issuecomment-26650102
-RUN sed -i 's/^mesg n/tty -s \&\& mesg n/g' /root/.profile
-
-# Switch to a bash login shell to allow simple 'rvm' in RUN commands
-SHELL [ "/bin/bash", "-l", "-c" ]
-
-RUN rvm use ${RUBY_VER} --install --binary --fuzzy
-
-# Optional: child images can change to this user, or add 'rvm' group to other user
-# see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
-RUN useradd -m --no-log-init -r -g rvm ${RVM_USER}
-
-RUN mkdir -p /metanorma
-COPY metanorma/Gemfile metanorma/Gemfile.lock /metanorma/
+# install metanorma toolchain
+RUN mkdir -p /setup
+COPY ${METANORMA_IMAGE_NAME}/Gemfile /setup/Gemfile
+RUN cd /setup && bundle install
 
-WORKDIR /metanorma
-RUN which ruby
-RUN rvm use ${RUBY_VER}; \
-  gem update --system; \
-  gem install bundler \
-  && gem install rake \
-  && gem install sassc \
-  && bundle
-
-# Add user "metanorma" as non-root user w/ sudo privileges
-RUN echo 'metanorma ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
-
-VOLUME /metanorma
-
-# USER ${RVM_USER}
+# export java executable path
+ENV JAVA_HOME /usr/lib/jvm/default-java
+ENV PATH $PATH:$JAVA_HOME/bin
 
+# cleanup and set base entrypoint
 RUN apt-get clean && rm -rf /var/lib/apt/lists/*
 
-ENTRYPOINT [ "/bin/bash", "-c", "-l" ]
```

Comment:
@abunashir have you tried using arguments to the `metanorma` command, such as:
* `metanorma compile -x html blah.adoc`
* `metanorma new -t ietf -d rfc my-new-doc`
?

Just wanted to make sure it works. We could add these to the `test` Makefile target. Thanks!

Reply:
Yap, it seems to be working fine.

<img width="1192" alt="Screenshot 2019-07-15 17 00 00" src="https://user-images.githubusercontent.com/1220911/61226225-52b79380-a722-11e9-9f86-aabb183bd911.png">

Reply:
Thanks @abunashir !